tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -426,6 +453,10 @@ func getOperatorLogs(runner *kubectlcmd.Runner, config *config.BugReportConfig,
 func getLog(runner *kubectlcmd.Runner, resources *cluster2.Resources, config *config.BugReportConfig,
 	namespace, pod, container string,
 ) (string, *processlog.Stats, int, error) {
+	defer func(start time.Time) {
+		log.Infof("Time used to get logs only for %s/%s/%s is %v", namespace, pod, container, time.Since(start))
```

Comment:
nit: it may be a bit clearer/less error prone to make a small helper function:

```

func trackTime(args ...any) (time.Time, []any) {
	return time.Now(), args
}

func logRuntime(start time.Time, args []any) {
	log.WithLabels("runtime", time.Since(start)).Infof(args...)
}
```

then can use like

```go
defer logRuntime(trackTime("get logs for %v/%v/%v", namespace, pod, container))
```

There might be a way to do it without the 2 functions but I am not sure how